javascriptv3/example_code/bedrock-agent/actions/list-agent-action-groups.js

```diff
@@ -0,0 +1,115 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import { fileURLToPath } from 'url';
+import { checkForPlaceholders } from "../lib/utils.js";
+
+import {
+    BedrockAgentClient,
+    ListAgentActionGroupsCommand,
+    paginateListAgentActionGroups
+} from '@aws-sdk/client-bedrock-agent';
+
+/**
+ * @typedef {Object} ActionGroupSummary
+ */
+
+/**
+ * Retrieves a list of Action Groups of an agent utilizing the ListAgentActionGroupsCommand.
+ *
+ * This function demonstrates the manual approach, sending a command to the client and processing the response.
+ * Pagination must manually be managed. For a simplified approach that abstracts away pagination logic, see
+ * the `listAgentActionGroupsWithPaginator()` example below.
+ *
+ * @param {string} agentId - The unique identifier of the agent.
+ * @param {string} agentVersion - The version of the agent.
+ * @param {string} [region='us-east-1'] - The AWS region in use.
+ * @returns {Promise<ActionGroupSummary[]>} A promise that resolves to an array of action group summaries.
+ */
+export const listAgentActionGroupsWithCommandObject = async (agentId, agentVersion, region = 'us-east-1') => {
```

Comment:
I've been favoring the paginators over the base commands. Including both seems good, but I'd give the paginator the higher listing.

Reply:
Done, in list-agents and in list-agent-action-groups